Src/Support/Google.Apis.Auth/OAuth2/GoogleCredential.cs

```diff
@@ -238,6 +239,11 @@ Task<string> ITokenAccess.GetAccessTokenForRequestAsync(string authUri, Cancella
             return credential.GetAccessTokenForRequestAsync(authUri, cancellationToken);
         }
 
+        IReadOnlyDictionary<string, string> ITokenAccessWithExtraInformation.ExtraInformation =>
+            (credential is ITokenAccessWithExtraInformation credentialWithExtra) ?
+            credentialWithExtra.ExtraInformation :
+            new ReadOnlyDictionary<string, string>(new Dictionary<string, string>());
```

Comment:
We should probably use a cached empty read-only dictionary. I know that can come later.